text/0000-argument-spread-splarguments.md

```diff
@@ -0,0 +1,81 @@
+- Start Date: 2019-06-05
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Forwarding Component Arguments with "Splarguments"
+
+## Summary
+
+Currently it is possible to forward element attributes and modifiers(?) into child components for use as "splattributes" 
+such as, `...attributes`. This RFC proposes adding similar syntax and functionality for arguments such as, `...arguments`. 
+
+## Motivation
+
+The reason for doing this is to allow less verbose component templates where the verbosity only makes the template harder to read.
```

Comment:
So I tend to be a bit more on the formal side here, but I think of the summary and the motivation as independent sections. Right now it reads like the summary is the first part of this section - if you remove it, this sentence doesn't make sense any more.

For instance, you could replace this with:

"Spreadable arguments will allow for less verbose component templates, especially where the verbosity..."

And then the sentence (and the section) stand on their own a bit more. Another option would be to start with the current state of the world:

"Currently, users must do x, y, and z in their applications. This is bad for reasons. Spreadable arguments would allow users to solve these problems in a better way..."

Reply:
How does it feel now?